tests/models/gpt_bigcode/test_modeling_gpt_bigcode.py

```diff
@@ -420,11 +420,6 @@ class GPTBigCodeMQAModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTe
         else ()
     )
     all_generative_model_classes = (GPTBigCodeForCausalLM,) if is_torch_available() else ()
-    fx_compatible = False
-    test_missing_keys = False
-    test_pruning = False
-    test_torchscript = False
-    multi_query = True
```

Comment:
just change the order. Not really matter, it's just the order generated by the script